src/views/Profile/AdminTable/AdminDatePicker.tsx

```diff
@@ -54,39 +132,51 @@ export const AdminDatePicker: React.FC<IMaterialsDate> = ({
           value={[]}
           disableUnderline
           displayEmpty
-          renderValue={() => (
-            <div>
-              {t(langTokens.admin.from)} {parsDate(start)}{' '}
-              {t(langTokens.admin.to)} {parsDate(end)}
-            </div>
-          )}
+          renderValue={() =>
+            !datePickerState.firstSelectedDay.value &&
+            !datePickerState.secondSelectedDay.value ? (
+              <DateRangeOutlinedIcon />
+            ) : (
+              <div>
+                {t(langTokens.admin.filterByDatePeriod, {
+                  rangeStart: parsDate(start),
+                  rangeEnd: parsDate(end),
+                })}
+              </div>
+            )
+          }
           MenuProps={MenuProps}
         >
           <Box display="flex" className={classes.pickersWrapper}>
-            <DatePicker
-              autoOk
-              orientation="portrait"
-              variant="static"
-              openTo="date"
-              value={start}
-              onChange={(value) => {
-                handleDatePicker('start', value);
-              }}
-              format="dd-mm-yyyy"
-              maxDate={end}
-            />
-            <DatePicker
-              autoOk
-              orientation="portrait"
-              variant="static"
-              openTo="date"
-              value={end}
-              onChange={(value) => {
-                handleDatePicker('end', value);
-              }}
-              format="dd-mm-yyyy"
-              minDate={start}
-            />
+            <Box className={datepickerClassesStart.datePicker}>
+              <DatePicker
+                autoOk
+                orientation="portrait"
+                variant="static"
+                openTo="date"
+                value={start}
+                onChange={(value) => {
+                  handleDatePicker('start', value);
+                }}
+                format="dd-mm-yyyy"
+                maxDate={today}
```

Comment:
how does code behave if we select `start` > `end`?

Reply:
We have one handleDatePicker and when we send two different flags - 'start', and 'end'. When we click on the first DatePicker we automatically send 'start', and when we click on the second DatePicker we automatically send 'end'. This logic was written before I started rewriting this component.

Reply:
let's me explain what I mean

say, today is 06/09

i'm setting end date through second picker to 31/08
now I'm setting start date through first picker to 02/09

then start date> end date

how will it behave in this case?

Reply:
so how does it work in such a case?

Reply:
I changed the logic in this component. And when you set the end date through the second picker to 31/08, now you can't set the start date through the first picker to 02/09. The start day must be earlier than the end day.